### PR TITLE
Add my card information

### PR DESCRIPTION
**Pull request by ksheersagaragrawal**
Added my card to index.html file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
...
+            <a href="https://www.twitter.com/ksheer_sagar" target="_blank">ksheer_sagar</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/ksheersagaragrawal" target="_blank">ksheersagaragrawal</a>
+          </p>
+          <p class="about">I'm a fresh grad from IIT Gandhinagar, EE + CS.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://www.freecodecamp.org/" target="_blank" title="First Resource">Twitter</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:138]:
```suggestion
                <a href="https://www.freecodecamp.org/" target="_blank" title="freeCodeCamp">freeCodeCamp</a>
```

File: index.html [hunk 2]
```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://leetcode.com" target="_blank" title="Second Resource">FreeCodeCamp</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:141]:
```suggestion
                <a href="https://leetcode.com" target="_blank" title="LeetCode">LeetCode</a>
```

File: index.html [hunk 3]
```diff
@@ -121,6 +121,33 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://github.com/explore" target="_blank" title="Third resource">Github explore</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:144]:
```suggestion
                <a href="https://github.com/explore" target="_blank" title="GitHub">Github explore</a>
```

**Review by StackOverflowIsBetterThanAnyAI — COMMENTED** [review]
You used the wrong link text for your resources.

**Review by CandidDeer — APPROVED** [review]

**Comment by CandidDeer** [comment]
Thank you for your contribution. I have merged your PR and your card will be live in a moment.

**Pull request merged**